web/src/typeahead_helper.ts

```diff
@@ -308,9 +304,7 @@ export function sort_people_for_relevance(
         current_stream = stream_data.get_sub_by_id(current_stream_id);
     }
     if (!current_stream) {
-        objs.sort((person_a, person_b) =>
-            compare_people_for_relevance(person_a, person_b, compare_by_pms),
-        );
+        objs.sort((person_a, person_b) => compare_people_for_relevance(person_a, person_b));
```

Comment:
Yeah, I guess a follow-up could be to add a `compare_by_current_conversation` for this case that literally just checks if the user is in the current DM conversation (if any), I suppose that would matter only in some rare/strange cases, like that they're a recipient of that DM conversation but haven't every sent a message you've seen?